src/main/resources/reference.conf

```diff
@@ -132,6 +132,10 @@ scorex {
 
     # Network controller timeout
     controllerTimeout = 5s
+
+    # How many persistent modifiers to store in the cache.
+    # The cache stores modifiers that are waiting to be applied.
+    maxModifiersCacheSize = 256
```

Comment:
Why this number? I think it should not be less then 500, because it will lead to synchronization problems when INV contains more ids, than maxModifiersCacheSize

Reply:
@catena2w increased to 512